docs/enterprise/orchestration.md

```diff
@@ -204,18 +221,202 @@ configs:
+### OpenID Connect
+
+As an example, the following `values` file will run lakeFS Enterprise with OIDC integration.
+
+
+**Note:** Full OIDC configurations explained [here][lakefs-sso-enterprise-spec-oidc].
+{: .note }
+
+1. Create `values.yaml` file and replace the placeholders with your OIDC provider details, token and ingress host.
+2. Run `helm install lakefs lakefs/lakefs -f values.yaml` in the desired K8S namespace.
+   
+```yaml
+lakefsConfig: |
+  logging:
+      level: "INFO"
+  blockstore:
+    type: s3
+  auth:
+    oidc:
+      # the claim that's provided by the OIDC provider (e.g Okta) that will be used as the username according to OIDC provider claims provided after successful authentication
+      friendly_name_claim_name: "<some-oidc-provider-claim-name>"
+      default_initial_groups: ["Developers", "Admins"]
+      # if true then the value of friendly_name_claim_name will be refreshed during each login to maintain the latest value
+      # and the the claim value (i.e user name) will be stored in the lakeFS database
+      persist_friendly_name: true
+    ui_config:
+      login_cookie_names:
+        - internal_auth_session
+        - oidc_auth_session
+ingress:
+  enabled: true
+  ingressClassName: <class-name>
+  hosts:
+    # the ingress that will be created for lakeFS
+    - host: <lakefs.acme.com>
+      paths: 
+       - /
+
+##################################################
+########### lakeFS enterprise - FLUFFY ###########
+##################################################
+
+fluffy:
+  enabled: true
+  image:
+    repository: treeverse/fluffy
+    pullPolicy: IfNotPresent
+    privateRegistry:
+      enabled: true
+      secretToken: <dockerhub-token-fluffy-image>
+  fluffyConfig: |
+    logging:
+      format: "json"
+      level: "INFO"
+    auth:
+      logout_redirect_url: https://oidc-provider-url.com/logout/example
+      oidc:
+        enabled: true
+        url: https://oidc-provider-url.com/
+        client_id: <oidc-client-id>
+        callback_base_url: https://<lakefs.acme.com>
+        # the claim name that represents the client identifier in the OIDC provider (e.g Okta)
+        logout_client_id_query_parameter: client_id
+        # the query parameters that will be used to redirect the user to the OIDC provider (e.g Okta) after logout
+        logout_endpoint_query_parameters:
+          - returnTo
+          - https://<lakefs.acme.com>/oidc/login
+  secrets:
+    create: true
+  sso:
+    enabled: true
+    oidc:
+      enabled: true
+      # secret given by the OIDC provider (e.g auth0, Okta, etc) store in kind: Secret
+      client_secret: <oidc-client-secret>
+  rbac:
+    enabled: true
+
+useDevPostgres: true
+```
+
+  </div>
+  <div markdown="1" id="saml">
+### SAML (Azure AD)
+
+The following example will walk you through the deployment of lakeFS Enterprise with SAML integration using Azure AD as the IDP. 
+Even though Azure Entra supports OIDC, the following example is using SAML on purpose.
```

Comment:
I get why it's confusing and open to suggestions, the reasoning is: 
1. We need some SAML example
2. Due to the nature of Microsoft servers, Azure enterprise users will be used to working with SAML protocols. 
3. Nevertheless more than once we had prospects struggling on how to create that SAML integration in Azure so that's why there's also step by step guide on how to configure it on azure since there are 100 ways but that one works.

Reply:
Thanks for explaining, then WDYT about: 
``` suggestion
The following example uses SAML, a common setup, although Azure Entra also supports OIDC.
```